src/analyses/goto_check.cpp

```diff
@@ -1931,9 +1932,7 @@ void goto_check(
   }
 }
 
-void goto_check(
-  const optionst &options,
-  goto_modelt &goto_model)
+void goto_check(const optionst &options, goto_modelt &goto_model)
```

Comment:
No change?

Reply:
Yes sorry (some reformatting got mixed in, because of doing some changes, reverting them, and applying clang-format meaning the commit and its revert don't cancel each other)